docs/user/security/authorization/index.asciidoc

```diff
@@ -95,9 +97,9 @@ image::user/security/images/create-role-dls-example.png[Create role with DLS ind
 To assign {kib} privileges to the role, click **Add {kib} privilege** in the {kib} section.
 
 [role="screenshot"]
-image::user/security/images/add-space-privileges.png[Add {kib} privileges]
+image::spaces/images/spaces-roles.png[Add {kib} privileges]
 
-Open the **Spaces** selection control to specify whether to grant the role access to all spaces *** Global (all spaces)** or one or more individual spaces. If you select *** Global (all spaces)**, you can’t select individual spaces until you clear your selection.
+Open the **Spaces** selection control to specify whether to grant the role access to all spaces *** All Spaces** or one or more individual spaces. If you select *** All Spaces**, you can’t select individual spaces until you clear your selection.
```

Comment:
Changed this and more text below to match what now shows in the UI -- we haven't displayed "Global (all spaces)" for a while.